wallet_connect/__init__.py

```diff
@@ -142,15 +140,15 @@ def get_redis_master(app):
     return web.json_response(error_message("Error unknown"), status=500)
 
 
-@routes.post('/update-transaction-status')
-async def update_transaction_status(request):
+@routes.post('/session/{sessionId}/transaction/{transactionId}/status/new')
```

Comment:
No need to put `/new` here as status is only part of the transaction. But, that's okay.